mage/how-to-guides/run-mage-modules-on-subgraphs.md

```diff
@@ -0,0 +1,139 @@
+---
+id: run-a-subgraph-module
+title: How to run a MAGE module on subgraphs
+sidebar_label: Run modules on subgraphs
+---
+
+If you have already read our how-to guide on [how to run MAGE modules and graph algorithms](/mage/how-to-guides/run-a-query-module.md),
+now it's time to move on to the finer execution of graph analytics.
+In this how-to guide, we will project a portion of the graph from the whole network persisted in 
+Memgraph, and run our algorithms on that portion of the graph.
+
+### Why might running an algorithm across the entire network not be ideal for you?
+
+If you didn't observe already, by executing a query module (or a MAGE module, which is essentially
+the same thing), the algorithm is executed on the whole network. This is impractical for multiple
+reasons:
+- your graph may be heterogeneous, and you want the module to run only on specific labels.
+- your graph may be too large, and you only want a portion of it to be updated with the analytics
+- you have multiple diverse data models and graphs in your network, and
+running analytics on all the graphs at the same time just does not make sense
+
+For that purpose, Memgraph has enabled you to execute modules on subgraphs and graph
+projections, to make the insights yielded by graph algorithms affect only the
+necessary nodes in your graph, saving you time and making the data more consistent
+and up to its specifications.
+
+### Available graph projections
+
+For now, the only graph projection function in Memgraph is called [project()](/cypher-manual/functions#graph-projection-functions),
+which can be carried out like as shown below: 
+
+```cypher
+MATCH p=(n)-[r]->(m)
+WITH project(p) AS graph
+RETURN graph;
+```
+
+Because we haven't specified any constraints, the result of this query
+is a projection of the graph that yields the entire graph.
+However, by putting constraints on properties or relationships, like in the query below:
```

Comment:
```suggestion
Because the matched pattern actually includes all the nodes and the relationships in the graph, the result of this query is a projection of the whole graph. 

To isolate a certain part of the graph, constraints need to be added to either properties or relationships, like in the query below: 
```

Reply:
I'm not sure if i wrote the second part of the first sentence correctly